src/System.Management.Automation/engine/InitialSessionState.cs

```diff
@@ -4421,6 +4423,14 @@ static InitialSessionState()
                     DefaultFormatEnumerationLimit,
                     RunspaceInit.FormatEnumerationLimitDescription),
 
+                // variable for PSModuleAutoLoadingPreference
+                new SessionStateVariableEntry(
```

Comment:
To avoid the breaking change, can't we have this outside of the list creation and check the env var before creating the default variable?